src/App.css

```
@@ -0,0 +1,78 @@
+.logoImg {
+    border-radius: 20px;
+    max-width: 200px;
+    margin-top: 15px;
+}
+
+.container {
+    max-width: 100vw;
```

Se o usuario usar uma tela de 50 polegadas, os seus componentes podem ficar muito grandes usando essa dimensao, é sempre bom ter um max-width com tamanho fixo, por exemplo 1400px.

```suggestion
    max-width: 1400px;
```